Core/Containers/DynamicsSelector.hpp

```diff
@@ -68,6 +69,11 @@ class DynamicsSelector : public DynamicsBase<
     [[nodiscard]] bool DynamicsArePaused(void) const override{
         return GetActiveModel()->DynamicsArePaused();
     }
+    // Calling GetPosition returns the quaternion as a Vector4
+    [[nodiscard]] inline Quaternion GetOrientation(void) const
+    {
+        return GetActiveModel()->GetOrientation();
+    }
```

Comment:
Please see comment in `DynamicsBase`.

Reply:
The GetOrientation function has been removed from DynamicsSelector